src/parachain/issue.ts

```diff
@@ -113,40 +140,100 @@ export class DefaultIssueAPI implements IssueAPI {
         this.transaction = new Transaction(api);
     }
 
+    async getRequestLimits(): Promise<IssueLimits> {
+        const vaults = await this.vaultsAPI.getVaultsWithIssuableTokens();
+        const [singleVault, maxTotal] = [...vaults.entries()].reduce(([singleVault, maxTotal], [_, vaultAvailable]) => {
```

Comment:
If I understand this reducer correctly, more suggestive would be `[singleVaultMaxIssuable, totalIssuable]` instead of `[singleVault, maxTotal]`